ci/deps/actions-37-db-min.yaml

```diff
@@ -31,7 +31,8 @@ dependencies:
   - openpyxl
   - pandas-gbq
   - google-cloud-bigquery>=1.27.2 # GH 36436
-  - pyarrow=0.17 # GH 38803
+  - protobuf>=3.12.4
+  - pyarrow=0.17.1 # GH 38803
```

Comment:
Using `pyarrow=0.17.0` would pin `protobuf` version and cause an AttributeError in gbq test.